src/io/flutter/editor/FlutterIconLineMarkerProvider.java

```diff
@@ -44,6 +62,18 @@
     PsiElement parent = refExpr.getParent();
     if (parent == null) return null;
 
+    // Resolve the class reference and check that it is one of the known, cached classes.
+    final PsiElement symbol = "IconData".equals(name) ? refExpr : refExpr.getFirstChild();
+    final PsiElement result = ((DartReference)symbol).resolve();
+    if (result == null) return null;
+    final List<VirtualFile> library = DartResolveUtil.findLibrary(result.getContainingFile());
+    if (library.size() > 1) return null;
```

Comment:
Is library size potentially 0?

Reply:
Yes, apparently. I changed to a for-loop to check all possibilities.